supabase/migrations/20241127203353_searchable_index_col.sql

```diff
@@ -0,0 +1,36 @@
+ALTER TABLE topics
+    ADD COLUMN searchable_index_col tsvector;
+
+CREATE INDEX search_idx ON topics USING gin(searchable_index_col);
```

Comment:
Is `gin` the preferred index type for tsvector columns?

Reply:
Yes, I googled and seems that gin is preferred for tsvector type.